msteams-platform/toolkit/create-new-teams-app-for-Visual-Studio.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Create new Teams app in Visual Studio
+author: surbhigupta
+description: In this module, learn how to create a new Teams app using Teams Toolkit for Visual Studio
+ms.author: v-amprasad
+ms.localizationpriority: high
+ms.topic: overview
+ms.date: 07/29/2022
+---
+# Create new Teams app in Visual Studio
+
+Teams Toolkit provides Microsoft Teams app templates in Visual Studio to create Teams app.  You can search and select Teams app template that you require when you create a new project. You can have Teams app templates for creating.
+
+* Tab app.
+* Command bot.
+* Notification bot.
+* Message extension.
+
+## Prerequisites
+
+| &nbsp; | Install | For using... |
+| --- | --- | --- |
+| &nbsp; | **Required** | &nbsp; |
+| &nbsp; | Visual Studio | You can install the free community edition of Visual Studio, and install the workload. |
```

Comment:
Visual Studio 2022 v17.3

Reply:
You can install the enterprise edition of Visual Studio, and install the "ASP.NET "workload and Microsoft Teams Development Tools.

Reply:
These comments have been implemented.